SQL/New_patches/2025-04-14_add-use-legacy-dicom-importer-config.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber)
+  SELECT 'use_legacy_dicom_study_importer', 'Use the legacy DICOM study importer instead of the new one.', 1, 0, 'boolean', ID, 'Use legacy DICOM study importer', 11
+  FROM ConfigSettings
+  WHERE Name="imaging_pipeline";
+
+INSERT INTO Config (ConfigID, Value)
+  SELECT ID, 0
```

Comment:
This should be set to 1 for existing projects that will be running this patch.

It can be set to 0 in the default schema those to favour the new DICOM importer.